src/aac/read.rs

```diff
@@ -102,10 +102,8 @@ where
 	}
 
 	if parse_options.read_properties {
-		let mut first_frame_header = match first_frame_header {
-			Some(header) => header,
-			// The search for sync bits was unsuccessful
-			None => decode_err!(@BAIL Mpeg, "File contains an invalid frame"),
+		let Some(mut first_frame_header) = first_frame_header else {
+			decode_err!(@BAIL Mpeg, "File contains an invalid frame");
 		};
```

Comment:
Does `manual_let_else` remove comments in its suggestion?

Reply:
Manual fix + human off-by-one error.